example/src/screens/Meeting/Modals.tsx

```diff
@@ -452,9 +460,18 @@ const ParticipantFilter = ({
   setFilter: React.Dispatch<React.SetStateAction<string>>;
 }) => {
   const [visible, setVisible] = useState<boolean>(false);
+  const [roles, setRoles] = useState<HMSRole[]>();
 
   const hideMenu = () => setVisible(false);
   const showMenu = () => setVisible(true);
+  const updateRoles = async () => {
+    setRoles(await instance?.getRoles());
+  };
+
+  useEffect(() => {
+    updateRoles();
```

Comment:
New roles will never be added while a room is ongoing. Rooms info once available from `onJoin` should be reused